app/models/medium.rb

```diff
@@ -202,7 +202,7 @@ def self.select_sorts
   end
 
   def self.advanced_sorts
-    ['RandomQuiz', 'Question', 'Remark', 'Erdbeere']
+    ['Question', 'Remark', 'Erdbeere']
```

Comment:
> In the 'Media Search' Tab that is available to editors in the admin area, it is now also possible to select media of type  Questions/Remark/Erdbeere. (This restores previous behaviour)

Yet, this has been possible beforehand as well, right? It's just that it is now also possible for _editors_, not just _admins_, right?

Reply:
It was possible before #420 for editors and admins to select "advanced sorts". In the current code in production, only admins can do that (by the line above that I removed in `media_helper.rb` and which is responsible for providing the sorts you can select in the dropdown). This change here is to make the distinction in `medium.rb` in lines 311 and 312 possible: 'Random Quizzes' are alway to be filtered out (I will explain this below), while the other sorts are filtered out on some condition.